contracts/Core/RewardManager.sol

```diff
@@ -126,7 +126,7 @@ contract RewardManager is Initializable, ACL, Constants {
         if (mediansLastEpoch.length == 0) return;
         uint64 penalty = 0;
         for (uint8 i = 0; i < mediansLastEpoch.length; i++) {
-            uint32 voteValueLastEpoch = voteManager.getVoteValue(i, stakerId);
+            uint32 voteValueLastEpoch = voteManager.getVoteValue(i + 1, stakerId);
```

Comment:
The only issue of solving it in this way is that if we decide to deactivate an asset, it will start pointing to the wrong assetID again.

Reply:
Let's tackle this in a separate issue. cc: @abhishekvispute @hrishikeshio @SamAg19